ql/src/test/queries/clientpositive/mytest.q

```diff
@@ -0,0 +1,85 @@
+add jar /home/msydoron/eclipse-workspace/JethroDataJDBCDriver/target/jethro-jdbc-3.6-standalone.jar;
```

Comment:
We will need tests to make sure that everything is working properly. As I assume we will not be able to have include tests against Jethro, I think we should at least test against Derby in the q files.